receiver/ReceiverTest.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import receiver.ReceiverAction;
+
+public class ReceiverTest {

```

Comment:
These are only testing the exception conditions. What about the logic of computing the statistics?